wp/headless-wp/includes/classes/Links.php

```diff
@@ -144,7 +144,7 @@ public function maybe_redirect_frontend() {
 			}
 
 			// Redirect the frontend WordPress request to the React website URL.
-			\wp_redirect( trailingslashit( esc_url_raw( $site_url ) ) . $url_request, 301 );
+			\wp_safe_redirect( trailingslashit( esc_url_raw( $site_url ) ) . $url_request, 301 );
```

Comment:
In order to move to `wp_safe_redirect` we would need to add `$site_url` to `allowed_redirect_hosts`.

Is there any meaninful benefit in switching to wp_safe_redirect here?

Reply:
I'm switching back to wp_redirect for now to get this unblocked but feel free to submit a followup PR if you believe there's a real benefit in using safe_redirect here.